spec/models/agent_spec.rb

```diff
@@ -1063,8 +1063,8 @@ def interpolate(string, agent)
 
   it 'should have .url' do
     t = '{{ agent.url }}'
-    expect(interpolate(t, @wsa1)).to eq("http://localhost/agents/#{@wsa1.id}")
-    expect(interpolate(t, @wsa2)).to eq("http://localhost/agents/#{@wsa2.id}")
-    expect(interpolate(t, @efa)).to  eq("http://localhost/agents/#{@efa.id}")
+    expect(interpolate(t, @wsa1)).to match(/http:\/\/localhost(?::\d+)?\/agents\/#{@wsa1.id}/)
```

Comment:
Maybe we can use `ENV['DOMAIN']` instead of `localhost`?

Reply:
Updated!

Reply:
I think `DOMAIN` also includes the port 😊

Reply:
Whoops, indeed it does!